### PR TITLE
jwt access token encryption decryption

### LINKED ISSUE #13: [Bug]: Github Access tokens are visible using https://jwt.io/
### Preflight Checklist

- [X] I have read the Contribution.md for this project.
- [X] I agree to follow the Code of Conduct that this project adheres to.
- [X] I have searched the issue for a feature request that matches the one I want to file, without success.

### Version

0.0.1

### Current Behavior

If I try to decode the user-token using [JWT Decode](https://jwt.io/) . 

I get user's public data but along with it I also get the third-party Oauth Token.
This could cause potential vulnerability.

### Expected Behavior

Third-party Oauth Token should be stored in a encrypted way and can be decrypted while validating user-token.

### Additional Information

You may use https://www.npmjs.com/package/aes256

### PR DESCRIPTION
**Pull request by MadhusreeBera**
# Description:

Jwt access token that was not encrypted previously and was a security issue has been resolved. 

Fixes #13 

## Type of change:

-   [X] Bug fix (non-breaking change which fixes an issue)

# Checklist:

-   [X] My code follows the style guidelines of this project.
-   [X] I have performed a self-review of my own code.
-   [X] My changes generate no new warnings.
-   [X] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -19,3 +19,5 @@ GITLAB_CLIENT_SECRET=XX
 
 RECAPTCHA_SECRECT_KEY=XX
 VALIDATE_RECAPTCHA=YES
+
+ENCRYPTION_SECRET_KEY=XX
```
Comment (Shubham-Kumar-2000) [inline sample.env]: Add a blank line

File: sample.test.env [hunk 2]
```diff
@@ -1 +1 @@
-TEST=YES
```
Comment (Shubham-Kumar-2000) [inline sample.test.env:1]: Revert this and run:
```
npm run quality
```
before committing.

File: src/helpers/jwt.helper.js [hunk 3]
```diff
@@ -27,6 +28,7 @@ exports.validate = (req, res, next) => {
         exports
             .verify(token)
             .then((user) => {
+                user.accessToken = decrypt(user.accessToken);
```
Comment (Shubham-Kumar-2000) [inline src/helpers/jwt.helper.js:31]: What happens if decrypt fails?

**Review by Shubham-Kumar-2000 — CHANGES_REQUESTED** [review]

**Comment by Shubham-Kumar-2000** [comment]
@MadhusreeBera  please update the `PR` with the required changes.

**Comment by MadhusreeBera** [comment]
I have made the required changes. Please check @Shubham-Kumar-2000

**Review by Shubham-Kumar-2000 — CHANGES_REQUESTED** [review]
Add test cases for `crypto.helper.js`

**Review by Shubham-Kumar-2000 — APPROVED** [review]

**Pull request merged**